packages/core/src/App/Constants/routes-config.js

```diff
@@ -293,6 +300,12 @@ const getModules = () => {
             is_authenticated: true,
             getTitle: () => localize('Account V2'),
         },
+        {
+            path: routes.cashier_v2,
+            component: Cashier_V2,
+            is_authenticated: true,
+            getTitle: () => localize('Cashier V2'),
```

Comment:
very tiny nitpick: I know we're stil in development, but ultimately, this is gonna be client-facing stuff eventually, so probably V2 not needed,

Reply:
@wojciech-deriv sorted